pkgs/applications/science/electronics/nanovna-saver/default.nix

```diff
@@ -4,16 +4,28 @@
   python3,
   fetchFromGitHub,
   qt6,
+  writeShellScriptBin,
 }:
+let
+  # Matches the pyside6-uic and pyside6-rcc implementations
+  # https://code.qt.io/cgit/pyside/pyside-setup.git/tree/sources/pyside-tools/pyside_tool.py?id=9b310d4c0654a244147766e382834b5e8bdeb762#n90
+  pyside-tools-uic = writeShellScriptBin "pyside6-uic" ''
+    exec ${qt6.qtbase}/libexec/uic -g python "$@"
+  '';
```

Comment:
This helper script is duplicated in [openusd](https://github.com/NixOS/nixpkgs/blob/0ab0fd44102fd7259708584c6eafb78b1aeee0d3/pkgs/development/python-modules/openusd/default.nix#L44-L50).
I think this minor duplication is okay and does not justify merging.
However, I am wondering why projects apparently expect pyside to provide this script, while our pyside doesn't include it. Is it because we are not building/including `pyside6-uic` on purpose for some reason?

Reply:
[`pyside2-tools`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pyside2-tools/default.nix) packs `uic` and `rcc` for Pyside 2. Though, I'm not quite certain if there are any packages that truly require these at build in the repo now.

I guess we can create a Pyside 6 version of this to reduce duplication between to packages